common/run/run.go

```diff
@@ -172,7 +174,10 @@ func Run() {
 
 	go pubsub.PollEvents()
 
-	common.RunCommonRunPlugins()
+	// Run common plugins for non-bot components
+	if !flagRunBot && !flagRunEverything {
+		common.RunCommonRunPlugins()

```

Comment:
Even if we would make this change, I'm not convinced that making the initialization order flag-dependent is a good idea at all.